### DISCUSSION TITLE
i have a problem

### DISCUSSION THREAD

**Discussion by iAmZam0** [Verwex/Funkin-Mic-d-Up-SC#60] [General]
hi,I've been trying to extract the file for days and hours but when I extract it it tells me that it is damaged, well I extracted it with 7-Zip,and when entering the folder there were no resources, docs, export there was only assets art and the .github folder, I came to ask if it is the file or an error please answer

**Comment by TentaRJ** [comment]
### 1. Please report these things to the Issues tab, not the Discussions tab.
### 2. Don't download the release zip from the Release tab, but rather the source code zip on the Code tab. You can download the current code [here](https://github.com/Verwex/Funkin-Mic-d-Up-SC/archive/refs/heads/main.zip)!

**Reply by iAmZam0** [reply]
ok thanks but that is what I wanted to talk about any file that I install of the code says that it is damaged and does not extract the source folder

**Reply by ghost** [reply]
thats just github messing up

**Reply by ArdaBeyNotFound** [reply]
Download from another browser

**Reply by ghost** [reply]
> Download from another browser

i dont think thst will help
